quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -40,12 +40,12 @@ function _wait_for_controller {
 }
 
 function _wait_for_public_router {
-  local advertised_host_port="${ZITI_ROUTER_NAME}:${ZITI_ROUTER_PORT}"
+  local advertised_host_port="${ZITI_ROUTER_ADVERTISED_ADDRESS}:${ZITI_ROUTER_PORT}"
```

Comment:
this only worked when the entity name and domain name of the router happened to match